index.html

```diff
@@ -40,9 +44,179 @@ <h1 class="description">I am Desmond, glad to see you!
           <li><a href="#"><img src=" ./images/M-icon.svg" alt="M-icon"></a> </li>
           <li><a href="#"><img src=" ./images/hand-raise.svg" alt="handraise icon"></a></li>
         </ul>
-      </section>
-
-      
+      </section> 
     </main>
+    <!-- work section -->
+    <section class="work">
+        <div class="container">
+            <div class="workSection">
+                <!-- Card 1 section -->
+                <div class="card ">
+                    <img class="card-image" src="./images/skills-snapshot.png" alt="skills-snapshot">
+                    <div class="card-content">
+                        <h1 class="card-title">Tonic</h1>
```

Comment:
- [ ] It is good practice to have only one `<h1>` per page, and that should be for the main title. Kindly change this to any of h2 - h6 tag. You can read more [here](https://www.hobo-web.co.uk/headers/)